khmer/utils.py

```diff
@@ -55,6 +55,10 @@ def check_is_pair(record1, record2):
     elif lhs1 == lhs2 and rhs1.startswith('1:') and rhs2.startswith('2:'):
         return True
 
+    # handle @name seq/1
+    elif lhs1 == lhs2 and rhs1.endswith('/1') and rhs2.endswith('/2'):
```

Comment:
Because `lhs1 ==lhs2` will always be true, even if pairing is off, as that's the sample ID of the file.

Also, sorry I didn't look at this earlier :)